app/src/main/java/de/triology/universeadm/ConstraintViolationResponse.java

```diff
@@ -0,0 +1,9 @@
+package de.triology.universeadm;
+
+public class ConstraintViolationResponse {
+  public Constraint.ID[] constraints;
```

Comment:
SonarQube warns about a possible vulnerability of publicly editable fields.

Should the field `constraints` be `private` or `final`?

Reply:
I think this cannot be private because otherwise it would not be converted to json correct. Final should be possible.